playbook.yaml

```diff
@@ -0,0 +1,56 @@
+---
+- name: install and start todo app
+  hosts: frontend
+  remote_user: ec2-user
+  vars_prompt:
+    - name: secret_key
+      prompt: Flask secret key?
+      private: false
+    - name: trello_token
+      prompt: Trello token?
+      private: false
+    - name: trello_key
+      prompt: Trello key?
+      private: false
+  tasks:
+  - name: Install Git, Python3
+    ansible.builtin.yum:
+      name:
+      - git
+      - python3
+      state: present
+    become: true
+  - name: Install Poetry
+    ansible.builtin.shell:
+      cmd: curl -sSL https://install.python-poetry.org | python3 -
+      creates: /home/ec2-user/.local/bin/poetry
+  - name: Make App Directory
+    ansible.builtin.shell:
+      cmd: >
+        mkdir /opt/todoapp &&
+        chown ec2-user /opt/todoapp
+      creates: /opt/todoapp
+    become: true
```

Comment:
I'm a fan of using specific modules over commands where we sensibly can, e.g. we could use the ["file" module](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/file_module.html) to have this handled in a more declarative manner. It might seem niche, but for example if someone modified permissions manually on the folder, then this command would ignore it but I'd expect the file module to notice that discrepancy and resolve it for us